TestProject/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using MarsQA_Nunit.Models;
+using MarsQA_Nunit.Pages;
+using MarsQA_Nunit.Utilities;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsQA_Nunit.Tests
+{
+    [TestFixture]
+    public class CertificationTests : CommonDriver
+    {
+        Login login = new Login();
+        Certifications certification = new Certifications();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors